Mathlib/Topology/Algebra/InfiniteSum/Group.lean

```diff
@@ -397,3 +397,29 @@ theorem tprod_const [T2Space G] (a : G) : ∏' _ : β, a = a ^ (Nat.card β) :=
       simpa [multipliable_const_iff] using ha
 
 end IsTopologicalGroup
+
+section CommGroupWithZero
+variable {K : Type*} [CommGroupWithZero K] [TopologicalSpace K] [ContinuousMul K]
+
+open Finset in
+lemma Multipliable.congr_cofinite'
```

Comment:
should this be called something like `Multipliable.eventually_congr` or something with eventually in the name?

Reply:
This part of the PR got split off into #23803. The naming is based on an existing lemma called `Multipliable.congr_cofinite`.

Reply:
Fair enough, in my mind when trying to find these lemmas I'd search for something with eventually in the title, but if thats a pre-existing name then I guess its me that needs updating!